### PR TITLE
docs: trivial fix some typos

### PR DESCRIPTION
**Pull request by truongnh1992**
Although it is spelling mistakes, it might make an affects while reading.

Signed-off-by: Nguyen Hai Truong <truongnh@vn.fujitsu.com>

### CODE REVIEW

File: docs/releases.md [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 #### 4. Release kops to homebrew
 
-* Following the [documentation](development/homebrew.md) we must release a compatible homebrew formulae with the release.
```
Comment (rifelpet) [inline docs/releases.md]: I think this is actually the correct spelling

https://formulae.brew.sh/

File: docs/calico-v3.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ will install v3.
 ### etcd
 Due to the etcd v3 API being a requirement of Calico Version 3
 (when using etcd as the datastore) not all Kops installations will be
-upgradable to Calico V3. Installations using etcd v2 (or earlier) will need
```
Comment (mikesplain) [inline docs/calico-v3.md]: Not that it matters too much, but I believe both of these are correct.  Most sites say they can be used interchangeably, and there's a similiar debate with the word [upgradable](https://english.stackexchange.com/questions/56431/updatable-vs-updateable-which-is-correct) and the corresponding [ngrams](https://books.google.com/ngrams/graph?content=upgradable%2Cupgradeable&year_start=1800&year_end=2000&corpus=0&smoothing=3&share=&direct_url=t1%3B%2Cupgradable%3B%2Cc0%3B.t1%3B%2Cupgradeable%3B%2Cc0).

I think my preference goes to `upgradable`.

File: docs/releases.md (hunk 1 above)
Comment (mikesplain) [reply docs/releases.md]: Agreed on this.

**Pull request merged**